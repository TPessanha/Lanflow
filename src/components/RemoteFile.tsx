import React from "react";

export class RemoteFile extends React.Component {
    public render() {
        return (
            <div>
                <h1>
                    This is where you browse for files in your phone.
                </h1>
            </div>
        );
    }
}