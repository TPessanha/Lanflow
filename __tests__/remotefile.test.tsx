import React from "react";
import renderer from "react-test-renderer";
import RemoteFile from "../src/components/RemoteFile";

it("renders home correctly", () => {
	const tree = renderer
		.create(<RemoteFile />)
		.toJSON();
	expect(tree).toMatchSnapshot();
});
