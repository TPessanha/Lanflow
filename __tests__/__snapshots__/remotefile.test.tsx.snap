// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders home correctly 1`] = `
<div>
  <h1>
    This is where you browse for files in your phone.
  </h1>
</div>
`;
